### PR TITLE
AoA: Localization of HuntPrey

### PR DESCRIPTION
**Pull request by manfred7168**
Changed Dmiri Yoltosha and Duergar Slave Lord: Set RuleElement of Hunt Prey to Localization via re-en.json

### CODE REVIEW

File: static/lang/re-en.json [hunk 1]
```diff
@@ -467,6 +467,9 @@
                 "BittersweetDreamsCriticalFailure": "Critical Failure",
                 "BittersweetDreamsSuccessFailure": "Success or Failure"
             },
+            "DuergarSlaveLord": {
+                "FirstHitDamage": "First Hit on Hunted Prey of the round"
```
Comment (TikaelSol) [inline static/lang/re-en.json:471]: I think we have this exact note elsewhere, no?

**Comment by manfred7168** [comment]
On another monster, yes. So, do I need to combine these two to use the same Label ID?

Gesendet von Mail für Windows

Von: TikaelSol
Gesendet: Sonntag, 24. Juli 2022 16:29
An: foundryvtt/pf2e
Cc: Manni; Author
Betreff: Re: [foundryvtt/pf2e] AoA: Localization of HuntPrey (PR #3103)

@TikaelSol commented on this pull request.

In static/lang/re-en.json:
> @@ -467,6 +467,9 @@
                 "BittersweetDreamsCriticalFailure": "Critical Failure",
                 "BittersweetDreamsSuccessFailure": "Success or Failure"
             },
+            "DuergarSlaveLord": {
+                "FirstHitDamage": "First Hit on Hunted Prey of the round"
I think we have this exact note elsewhere, no?

**Comment by TikaelSol** [comment]
Yeah, before we do a release let's just make a generic NPCAbility section with HuntPrey under it and consolidate. No sense in duplicating text here.

**Comment by TikaelSol** [comment]
Oh, we already have hunt prey in there, just throw everything under that.

**Pull request closed without merging**